src/components/CommentList.jsx

```diff
@@ -0,0 +1,49 @@
+import { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { getCommentsByArticle } from "../api";
+import AddComment from "./AddComment";
+
+
+const CommentList = () => {
+
+    const {article_id} = useParams()
+    const [commentList, setCommentList] = useState([])
+    const [isLoading,setIsLoading] = useState(true)
+
+    useEffect(() => {
+        setIsLoading(true)
+        getCommentsByArticle(article_id).then(res => {
+            setCommentList(res.comments)
+            setIsLoading(false)
+        })
+    },[article_id])
+
+
+
+    if (isLoading) {
+        return <p>Loading ...</p>
+    }
+
+    return (
+        <div className="CommentList">
```

Comment:
Could we find another HTML Tag to use to contain the list of comments that has more meaning to a screen reader?

Reply:
not sure which tag would be appropriate